spec/features/user_signs_out_spec.rb

```diff
@@ -10,21 +10,21 @@
   # * When I opt to sign out, I get a confirmation that my identity has been
   #   forgotten on the machine I'm using
 
-  # scenario 'authenticated user signs out' do
-    # user = FactoryGirl.create(:user)
+  scenario 'authenticated user signs out' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.